streams-util/src/main/java/org/apache/streams/util/schema/SchemaStoreImpl.java

```diff
@@ -63,7 +63,7 @@ public synchronized Schema create(URI uri) {
       if (uri.toString().contains("#") && !uri.toString().endsWith("#")) {
         Schema newSchema = new Schema(baseUri, baseNode, null, true);
         this.schemas.put(baseUri, newSchema);
-        JsonNode childContent = this.fragmentResolver.resolve(baseNode, '#' + StringUtils.substringAfter(uri.toString(), "#"));
+        JsonNode childContent = this.fragmentResolver.resolve(baseNode, '#' + StringUtils.substringAfter(uri.toString(), "#"), "#");
```

Comment:
What's going on here? Is this change related to the `jsonschema2pojo` version bump, or something else?

Reply:
I'm not entirely sure - I recall I was working through failing builds and/or tests that were repaired with this change.